standalone/src/models/store/store/store-draft/presets/empty.spec.ts

```diff
@@ -1,8 +1,39 @@
-import { TStoreDraft } from '../../types';
-import empty from './empty';
+import { TStoreDraftGraphql, TStoreDraftRest } from '../../types';
+import { emptyDraftRest, emptyDraftGraphql, empty } from './empty';
 
 it('should set all specified fields to undefined', () => {
-  const emptyStoreDraft = empty().build<TStoreDraft>();
+  const emptyStoreDraft = empty().build<TStoreDraftRest | TStoreDraftGraphql>();
+  expect(emptyStoreDraft).toMatchObject({
+    key: expect.any(String),
+    name: undefined,
+    languages: undefined,
+    countries: undefined,
+    distributionChannels: undefined,
+    supplyChannels: undefined,
+    productSelections: undefined,
+    custom: undefined,
+  });
+});
+
+it('should set all specified fields to undefined in graphql', () => {
+  const emptyStoreDraft =
+    emptyDraftGraphql().buildGraphql<TStoreDraftGraphql>();
+  expect(emptyStoreDraft).toMatchObject({
+    key: expect.any(String),
+    name: undefined,
+    languages: undefined,
+    countries: undefined,
+    distributionChannels: undefined,
+    supplyChannels: undefined,
+    productSelections: undefined,
+    custom: undefined,
+  });
+});
+
+it('should set all specified fields to undefined in rest', () => {
+  const emptyStoreDraft = emptyDraftRest().build<
+    TStoreDraftRest | TStoreDraftGraphql
+  >();
```

Comment:
No need for the types as we are already building from the rest version of the mode.
```suggestion
  const emptyStoreDraft = emptyDraftRest().build);
```

Reply:
here: [49ee1f9](https://github.com/commercetools/test-data/pull/836/commits/49ee1f96e438426b76a9ea4c06ee71b4943f38e8)